### PR TITLE
fix(generator-superset): add fixes to viz plugin generator

### PR DESCRIPTION
**Pull request by villebro**
🐛 Bug Fix
Fix viz plugin generator for breaking changes introduced by upgrade to `0.15`. Also do minor cleanup to types.

Verified to work by creating Hello World plugin via `npm link` on `incubator-superset` (no TypeScript errors), running `yarn build` and `yarn jest`.

### CODE REVIEW

File: packages/generator-superset/generators/plugin-chart/templates/src/types.erb [hunk 1]
```diff
@@ -0,0 +1,40 @@
+/**
```
Comment (rusackas) [inline packages/generator-superset/generators/plugin-chart/templates/src/types.erb:1]: Might just want to add this to the file tree in the README... or remove the file tree from the README (not sure how much value it really provides)

File: packages/generator-superset/generators/plugin-chart/templates/tsconfig.json [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+  ],
+  "references": [
+    {
+      "path": "../../packages/superset-ui-chart-controls"
+    },
+    {
+      "path": "../../packages/superset-ui-core"
+    }
+  ]
+}
```
Comment (rusackas) [inline packages/generator-superset/generators/plugin-chart/templates/tsconfig.json:25]: Hmm... surprised it's not barking about a newline here...

**Review by rusackas — APPROVED** [review]
Couple o' nits, but not seeing anything blocking! Thanks for cleaning this up!

File: packages/generator-superset/generators/plugin-chart/templates/src/types.erb (hunk 1 above)
Comment (villebro) [reply packages/generator-superset/generators/plugin-chart/templates/src/types.erb:1]: Good catch. `types.ts` was actually already in the README, but for some reason file was missing before this (probably fell out during the long development/review process). However, `tsconfig.json` was missing, so I added that.

File: packages/generator-superset/generators/plugin-chart/templates/tsconfig.json (hunk 2 above)
Comment (villebro) [reply packages/generator-superset/generators/plugin-chart/templates/tsconfig.json]: 👍  Added newline.

**Pull request merged**